src/Components/Publishing/Sections/ArtworkCaption.tsx

```diff
@@ -22,22 +22,15 @@ interface StyledArtworkCaptionProps {
 }
 
 export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
-  joinParts(children, delimiter = ', ') {
-    const joined = _.compact(children)
-      .reduce((prev, curr) => {
-        return [
-          prev,
-          delimiter,
-          curr
-        ]
-      })
+  joinParts(children, delimiter = ", ") {
+    const joined = _.compact(children).reduce((prev, curr) => {
+      return [prev, delimiter, curr]
+    }, [])
```

Comment:
Was missing `[]`.

Reply:
Actually, this is a bit stranger now that I'm looking at it closer. For some reason infering a start value leads to correct test results, but compacting an end value _with_ a start value errors out. Looking...

Reply:
Interesting: https://hackernoon.com/reduce-faster-with-no-starting-value-3d697c9464f9

Reply:
And `arr.reduce(callback[, initialValue])` is the signature -- [no starting value required](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)